testplan/testing/multitest/base.py

```diff
@@ -487,6 +487,7 @@ def pre_resource_steps(self):
                     label="before_start", func=self.cfg.before_start
                 )
             )
+        self._add_step(self.append_pre_post_step_report)
```

Comment:
if there is no before_start configured
nothing create the `self._pre_post_step_report`  so the `self.append_pre_post_step_report` will not append anything, if I reading this correctly.

Reply:
If this call is moved here, the docstring of `append_pre_post_step_report` is no more valid, needs update